src/node/rpc/node_frontend.h

```diff
@@ -821,6 +838,34 @@ namespace ccf
         .set_auto_schema<void, GetNetworkInfo::Out>()
         .install();
 
+      auto network_previous_identity = [this](auto& args, nlohmann::json&&) {
+        auto psi_handle = args.tx.template ro<ccf::PreviousServiceIdentity>(
+          ccf::Tables::PREVIOUS_SERVICE_IDENTITY);
+        const auto psi = psi_handle->get();
+        if (psi.has_value())
+        {
+          GetNetworkPreviousIdentity::Out out;
+          out.previous_service_identity = psi.value();
+          return make_success(out);
+        }
+        else
+        {
+          return make_error(
+            HTTP_STATUS_NOT_FOUND,
+            ccf::errors::ResourceNotFound,
+            "This service is not a recovery of a previous service.");
+        }
+      };
+      make_read_only_endpoint(
+        "/network/previous_identity",
```

Comment:
Should this be `/service/previous_identity`? Or maybe, should this be folded into the existing `GET /node/network` endpoint?

Reply:
Yes to the first! I thought we had multiple endpoints already nested under `/node/network`, but looking again they're mostly variations on `/node/network` itself. So this should probably be `/node/service/previous_identity`, similar to `/node/service/configuration`.